.github/workflows/terrascan.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: terrascan
+
+on:
+  workflow_call:
+    inputs:
+      iac_type:
+        required: true
+        type: string
+        default: terraform
```

Comment:
Lets add the options here, if we want to use inputs: helm, k8s, kustomize, terraform

Reply:
update description, no need to update worklfow if real workflow changes